runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBrowseView/Header.tsx

```diff
@@ -27,10 +29,36 @@ interface HeaderProps {
     className?: string;
 }
 
+enum ProcesTabs {
+    PROCESSES = 'processes',
+    COLLECTIONS = 'collections'
+}
+
 const Header: FC<HeaderProps> = ({ className, ...rest }) => {
     const { translate } = useTranslations();
     const hasProcessAddAccess = useFeatureKey([FeatureKey.PROCESS_ADD]);
     const hasAddCollectionAccess = useFeatureKey([FeatureKey.PROCESS_COLLECTION_ADD]);
+    
+    const router = useRouter();
+    const currentTab = router.asPath.split('/').slice(-1)[0].split('?')[0];
```

Comment:
to be clear the url we expect: 'http://localhost:3000/app/collections?page=3&pageSize=12&collectionId=abcd-efgh-ijkl-mnop'

tested, works :)